src/tree/tree.scss

```
@@ -0,0 +1,149 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+@import '../icon/index';
+
+$iui-active-outline: thin solid t(iui-color-foreground-primary);
+
+// Spacing for when expander button is not present: (icon margin 6px) + (icon width 16px) + (button padding 4px) + (button border 2px)
+$iui-expander-button-width: 28px;
```

```suggestion
// 28px = (icon margin) + (icon width) + (button padding) + (button border 2px)
$iui-expander-button-width: ($iui-component-offset * 2) + ($iui-icons-default) + ($iui-xxs * 2) + 2px;
```

---

You can also add a second variable to make things more readable/usable.
```suggestion
$iui-expander-inline-padding: $iui-component-offset + $iui-xxs + 1px; // margin + padding + border
$iui-expander-button-width: ($iui-icons-default) + ($iui-expander-inline-padding * 2); // icon width + inline padding
```

---

Changed